### DISCUSSION TITLE
SQL files not indexed

### DISCUSSION THREAD

**Discussion by michaeljmcd** [oracle/opengrok#4056] [Q&A]
I set up OpenGrok on my local machine following the steps at https://github.com/oracle/opengrok/wiki/How-to-setup-OpenGrok. The projects were mostly Java + SQL and the indexer completes without errors. For some reason, searches only match against Java files. SQL (and, come to think of it, pom.xml files) aren't searchable. 

This is with the project set correctly, attempting either full search or symbol and file type set to Any.

I've even tried navigating through the projects in the OpenGrok web interface to the SQL (which does show up) and clicking on a symbol, which matches nothing, not even the file to which I had navigated.

Is there some other setup I need to do to get files besides Java source indexed? I also tried indexing with the option `-A .sql:SQLAnalyzer` added and that didn't help.

**Comment by michaeljmcd** [comment]
After fiddling around a little more, I realized that any repository added to the project after the first index wasn't updated by subsequent indexes (despite rendering in the navigation). I deleted the `data` directory and reindexed and everything worked fine. I'm not sure if there was a more elegant way to solve that problem though.

**Reply by vladak** [reply]
How do you run the indexer ? (exact options)
